account_invoice_email_status/models/invoice.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+
+from openerp import api, fields, models
+
+
+class account_invoice(models.Model):
+    _inherit = 'account.invoice'
+
+    email_status = fields.Selection([
+        ('outgoing', 'Outgoing'),
+        ('sent', 'Sent'),
+        ('received', 'Received'),
+        ('exception', 'Delivery Failed'),
+        ('cancel', 'Cancelled'),
+    ], 'Email Status', compute='compute_email_status')
+
+    @api.multi
+    def compute_email_status(self):
+        for rec in self:
```

Comment:
`invoice` would be a better name for `rec` according to me.


> Mal nommer les choses c'est ajouter au malheur du monde.
> 
> \- Camus

Reply:
Il as raison. :)